### ISSUE TITLE
Delete Fake Wallet On Real Wallet Activation

### ISSUE THREAD

**Issue by amihaiemil** [self-xdsd/self-web#389]
When activating a real wallet, we should remove the fake wallet.

**Comment by zoeself** [comment]
@amihaiemil this is your task now, please go ahead. Deadline (when this ticket should be closed) is ``2021-03-15T06:33:08.195906``.

Estimation here is ``60 minutes``, that's how much you will be paid.

Remember, you don't have to solve everything in this ticket - you can solve it [partially](https://docs.self-xdsd.com/microtasking.html) and leave ``todo`` markers in the code, which will become future tasks.

If you have any questions don't ask me, I'm not a technical person. Open new tickets instead.

If you don't have time or simply don't want to work on this, you can always [resign](https://docs.self-xdsd.com/projectmanager.html#commands).

**Comment by zoeself** [comment]
@amihaiemil please keep in mind that manual assignment of tickets is a bad practice and it is discouraged. Next time, please let me elect the assignee.

**Comment by zoeself** [comment]
@amihaiemil Don't forget to close this ticket before the deadline (2021-03-15T06:33:08). You are past the first half of the allowed period.

**Comment by amihaiemil** [comment]
@zoeself quit

**Comment by zoeself** [comment]
> @zoeself quit

@amihaiemil ok, I'll find someone else to take care of it.

**Comment by amihaiemil** [comment]
@criske can you take care of it when you have some time?

The change in the front-end: when the real wallet is activated, delete the Fake Wallet div. 

The change in the backe-end: in the activate wallet endpoint, use method ``wallet.activate()`` instead of ``wallets.activate(Wallet)`` (the implementation of StripeWallet.activate() will remove the FakeWallet).

**Comment by zoeself** [comment]
@criske this is your task now, please go ahead. Deadline (when this ticket should be closed) is ``2021-03-24T12:37:17.739102``.

Estimation here is ``60 minutes``, that's how much you will be paid.

Remember, you don't have to solve everything in this ticket - you can solve it [partially](https://docs.self-xdsd.com/microtasking.html) and leave ``todo`` markers in the code, which will become future tasks.

If you have any questions don't ask me, I'm not a technical person. Open new tickets instead.

If you don't have time or simply don't want to work on this, you can always [resign](https://docs.self-xdsd.com/projectmanager.html#commands).

**Comment by amihaiemil** [comment]
@criske btw, we introduced an Invoice's payments. Make sure to run the following on your local DB:

```sql
ALTER TABLE `self_xdsd`.`slf_invoices_xdsd` DROP COLUMN `transactionId`, DROP COLUMN `payment_timestamp`;

DELETE FROM self_xdsd.slf_invoices_xdsd WHERE 1=1;

CREATE TABLE `self_xdsd`.`slf_payments_xdsd` (
  `invoiceId` INT NOT NULL,
  `transactionId` VARCHAR(256) NOT NULL,
  `payment_timestamp` DATETIME NOT NULL,
  `value` DECIMAL(20,0) NOT NULL,
  `status` VARCHAR(32) NOT NULL,
  `failReason` VARCHAR(512) NOT NULL,
  PRIMARY KEY (`invoiceId`, `transactionId`, `payment_timestamp`),
  CONSTRAINT `paidInvoiceFk`
    FOREIGN KEY (`invoiceId`)
    REFERENCES `self_xdsd`.`slf_invoices_xdsd` (`invoiceId`)
    ON DELETE CASCADE);

```

**Comment by criske** [comment]
waiting  for https://github.com/self-xdsd/self-core/pull/1061

**Comment by amihaiemil** [comment]
@zoeself remove

**Comment by zoeself** [comment]
> @zoeself remove

@amihaiemil ok, I've removed this task from scope. I'm not managing it anymore.

**Comment by amihaiemil** [comment]
@zoeself add

**Comment by zoeself** [comment]
> @zoeself add

@amihaiemil I've just registered this ticket as a task and will assign it to someone soon. Thanks!

**Comment by zoeself** [comment]
@criske this is your task now, please go ahead. Deadline (when this ticket should be closed) is ``2021-03-27T13:47:17.736350``.

Estimation here is ``120 minutes``, that's how much you will be paid.

Remember, you don't have to solve everything in this ticket - you can solve it [partially](https://docs.self-xdsd.com/microtasking.html) and leave ``todo`` markers in the code, which will become future tasks.

If you have any questions don't ask me, I'm not a technical person. Open new tickets instead.

If you don't have time or simply don't want to work on this, you can always [resign](https://docs.self-xdsd.com/projectmanager.html#commands).

**Comment by zoeself** [comment]
@criske thank you for resolving this ticket. I've just added it to your active invoice. You can always check all your invoices and more on the [Contributor Dashboard](https://self-xdsd.com).
